### ISSUE TITLE
[Crash] Game randomly crashes

### ISSUE THREAD

**Issue by ugabuga69** [sp614x/optifine#6864]
## Description of Issue
<Game randomly crashes, not a major issue but pretty annoying>

## OptiFine Version
<1.19 H8 pre1>

## Installation Method
<Normal instalation, by client>

## Fabric/Forge Version
<None>

## OptiFabric Version
<None>

## Other Installed Mods
<None>

**Comment by UnknownTheLad** [comment]
Please provide a crashlog

**Comment by ave118** [comment]
[crash-2022-06-29_12.29.02-client.txt](https://github.com/sp614x/optifine/files/9012495/crash-2022-06-29_12.29.02-client.txt)
[crash-2022-06-29_12.41.54-client.txt](https://github.com/sp614x/optifine/files/9012496/crash-2022-06-29_12.41.54-client.txt)
[crash-2022-06-29_12.45.22-client.txt](https://github.com/sp614x/optifine/files/9012497/crash-2022-06-29_12.45.22-client.txt)
[crash-2022-06-29_12.28.24-client.txt](https://github.com/sp614x/optifine/files/9012498/crash-2022-06-29_12.28.24-client.txt)
heres the 4 crashes i had

**Comment by UnknownTheLad** [comment]
Yep, wouldn't be surprised if Uga's crash is for the same reason. Guess we shall wait and see.

**Comment by KaiAF** [comment]
Reported #6858
